docs/src/pages/components/useTheme.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: useTheme
+group: Hooks
+caption: Hook that applies a theme
+github: https://github.com/marigold-ui/marigold/blob/main/packages/system/src/hooks/useTheme.tsx
+edit: https://github.com/marigold-ui/marigold/blob/main/docs/src/pages/components/useTheme.mdx
+switchTheme: true
+---
+
+The `useTheme` is a hook that gets you the current theme and all of its attributes and properties.
+
+The `useTheme` hook returns tree parameters:
+
+- **First:** returns the current `theme`, if there isn't a theme it uses the `__defaultThme`
```

Comment:
Props Table and list`theme``css` and `get` ? :)

Reply:
but are they props I thought its just the returns

Reply:
Yes they are 🤔 can we still use the same table somehow?

Reply:
I think no, because the PropsTable has a hard-coded header 🤔